### PR TITLE
8320675: PrinterJob/SecurityDialogTest.java hangs

### PR DESCRIPTION
**Pull request by luchenlin**
I backport this for parity with 17.0.13-oracle.
modify lines are different, but code is clean.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] [JDK-8320675](https://bugs.openjdk.org/browse/JDK-8320675) needs maintainer approval

### Issue
 * [JDK-8320675](https://bugs.openjdk.org/browse/JDK-8320675): PrinterJob/SecurityDialogTest.java hangs (**Bug** - P4 - Approved)

### Reviewers
 * [Paul Hohensee](https://openjdk.org/census#phh) (@phohensee - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk17u-dev.git pull/2807/head:pull/2807` \
`$ git checkout pull/2807`

Update a local copy of the PR: \
`$ git checkout pull/2807` \
`$ git pull https://git.openjdk.org/jdk17u-dev.git pull/2807/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 2807`

View PR using the GUI difftool: \
`$ git pr show -t 2807`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk17u-dev/pull/2807.diff">https://git.openjdk.org/jdk17u-dev/pull/2807.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk17u-dev/pull/2807#issuecomment-2290773912)

### CODE REVIEW

**Review by phohensee — APPROVED** [review]

**Comment by GoeLin** [comment]
Hi Andrew, 
they changed the review process. If a PR is modified after it has been reviewed, the review is invalidated and it need a new review.  I'll take a look.

**Comment by GoeLin** [comment]
Hi Andrew, 

can you please change the "/**" in line 38 to be "/*".
Also, please fix the indentation of the String list in INSTRUCTIONS
At the very end of the file a CR is missing.

Thanks, Goetz.

**Comment by luchenlin** [comment]
> Hi Andrew,
> 
> can you please change the "/**" in line 38 to be "/*". Also, please fix the indentation of the String list in INSTRUCTIONS At the very end of the file a CR is missing.
> 
> Thanks, Goetz.

hi @GoeLin , it's fixed, please check, thanks.

File: test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java [hunk 1]
```diff
@@ -21,210 +21,110 @@
...
+                    "\n" +
+                    "If the dialog has an option to save to file, the option ought " +
+                    "to be disabled.\n" +
+                    "\n" +
+                    "Press the Pass button if the \"Print to file\" option was disabled in\n" +
+                    "all the dialogs where it was present.\n" +
+                    "Otherwise, press the Fail button.\n" +
+                    "\n" +
+                    "The dialogs should be displayed even when \n" +
+                    "there is no queuePrintJob permission.";
```
[the remark this answers is not in the document]
Comment (luchenlin) [reply test/jdk/java/awt/print/PrinterJob/SecurityDialogTest.java]: @GoeLin  it's right now, thanks

**Review by phohensee — APPROVED** [review]

**Pull request closed without merging**